packages/api/src/jobs/save_page.ts

```diff
@@ -382,30 +337,45 @@ export const savePageJob = async (data: Data, attemptsMade: number) => {
       return true
     }
 
-    // for non-pdf content, we need to save the page
-    const apiResponse = await sendSavePageMutation(userId, {
-      url,
-      clientRequestId: articleSavingRequestId,
-      title,
-      originalContent: content,
-      parseResult: readabilityResult,
-      state,
-      labels,
-      rssFeedUrl,
-      savedAt,
-      publishedAt,
-      source,
-      folder,
-    })
-    if (!apiResponse) {
-      throw new Error('error while saving page')
+    if (!content) {
+      throw new Error(
+        'Invalid SavePage job, fetch result missing required data'
+      )
```

Comment:
probably have to remove this check

Reply:
I think we need it as we can't pass `originalContent` if its null.